### ISSUE TITLE
The ROS2 Urdf importer resolves packages incorrectly

### ISSUE THREAD

**Issue by michalpelka** [o3de/o3de-extras#225]
URDF/XACRO can contain such syntax:
```
        <geometry>
          <mesh filename="package://husky_description/meshes/base_link.dae" />
        </geometry>
```
It is resolved incorrectly :
```
/home/michal/ros2_workspace/husky/husky_description/husky_description/base_link.dae
```
That causes these meshes cannot be loaded.
![image](https://user-images.githubusercontent.com/3209244/226902623-e489346a-29cf-473f-aaca-ce27861b9838.png)

**Comment by michalpelka** [comment]
Beinging fixed #226

**Comment by michalpelka** [comment]
Fixed in #226
